layouts/_default/_markup/render-codeblock-plantuml.html

```diff
@@ -0,0 +1,18 @@
+{{- .Page.Scratch.Set "enablePlantUML" true -}}
+{{- $params := .Page.Site.Params -}}
+{{- $validFormat := slice "svg" "png" -}}
+{{- $server := $params.plantuml.server | default "https://www.plantuml.com/plantuml" -}}
+{{- $alt := index .Attributes "title" -}}
+{{- $format := index .Attributes "format" | default "svg" -}}
+
+{{- if in $validFormat $format | not -}}
+    {{- errorf "unsupported plantuml: %s!" $format -}}
+{{- end -}}
+
+<div id="plantuml-diagram">
+    <pre class='plantuml'
+        data-content="{{ .Inner | safeHTML }}"
+        data-server="{{ $server }}"
+        data-format="{{ $format }}"
+    />
```

Comment:
`pre` is a non-void HTML element. It needs to be closed by `<pre>data</pre>`.

Why use `pre` here? An empty `pre` tag has impact on the layout. Can we use a `script` tag just like WaveDrom?